spec/elastic_apm/transport/serializers/error_serializer_spec.rb

```diff
@@ -34,10 +34,65 @@ module Serializers
               code: nil,
               attributes: nil,
               stacktrace: be_an(Array),
-              handled: true
+              handled: true,
+              cause: nil
             )
           end
 
+          context 'with an exception chain', :mock_time do
+            it 'matches format' do
+              error = builder.build_exception(actual_chained_exception)
+              result = subject.build(error).fetch(:error)
+
+              expect(result).to include(
```

Comment:
Just a sidenote; do you know `match()`

```ruby
expect(result).to match(
  id: String,
  timestamp: 123,
  even_regex: /abc-\d{3}/
)
```

I don't have a preference. I didn't know one could do `be_a` inside `include` - handy!

Reply:
But I stole the test format from the existing ones in the file ; ) ([see master](https://github.com/elastic/apm-agent-ruby/blob/master/spec/elastic_apm/transport/serializers/error_serializer_spec.rb#L20-L28))
There were tests using `include` with `be_a` so I used that pattern. Is there a subtle difference between the tests I added and the existing ones that would make using `match` more relevant?